update-dois/src/test/java/org/reactome/release/updateDOIs/TestUpdateDOIs.java

```diff
@@ -49,7 +53,7 @@ public void testfindNewDOIsAndUpdate() throws Exception
 		Mockito.when(mockAdaptor.fetchInstanceByAttribute("Pathway", "doi", "NOT REGEXP", "^10.3180")).thenReturn(testResults);
 		Mockito.when(mockAdaptor.fetchInstanceByAttribute("Pathway", "DB_ID", "=", "67890")).thenReturn(testResults);
 
-		check.findAndUpdateDOIs(12345L, "reportPath", "70", true);
+		check.findAndUpdateDOIs(12345L, MOCK_REPORT_PATH, MOCK_RELEASE_NUMBER, true);
```

Comment:
Extracting `12345L` into a named constant would also be good for consistency, context, and re-use in the two times below.